kotlintest-core/build.gradle

```diff
@@ -1,5 +1,4 @@
 dependencies {
-    compile project(':kotlintest-assertions')
```

Comment:
Do we not want this now ?

Reply:
IDK if we want it, but we don't need it anymore.
We can add it, no difference to me, but it's not necessary

Reply:
I can add it back, no problem.
I just thought that maybe it's not a dependency to core anymore

Reply:
Why was it, I can't remember.

Reply:
Due to SysProps using something from core, if I remember correctly